docs/cli.rst

```diff
@@ -359,7 +359,7 @@ This subcommand has additional options:
    Choose a backend: local, ssh, winrm, docker.
 
 ``-c``, ``--command=COMMAND``
-
+   A single command string to run instead of launching the shell.
```

Comment:
This file will be generated by our rakescript, see https://github.com/chef/inspec/blob/master/Rakefile#L161-L210. Therefore we should add this information to https://github.com/chef/inspec/blob/master/lib/inspec/cli.rb#L157